docs/time/index.rst

```diff
@@ -364,7 +364,6 @@ local  Local Time Scale          (LOCAL)
 .. [#] Wikipedia `time standard <https://en.wikipedia.org/wiki/Time_standard>`_ article
 .. [#] SOFA_ Time Scale and Calendar Tools
        `(PDF) <http://www.iausofa.org/sofa_ts_c.pdf>`_
-.. [#] `<https://www.ucolick.org/~sla/leapsecs/timescales.html>`_
```

Comment:
You'd also need to remove the reference to this footnote on line 345 (which would become `both" [#]_, [#]_.`, I think.

Reply:
Ops. Thanks!